### PR TITLE
feat: implement create case studies API

### PR DESCRIPTION
**Pull request by Cosmoboi-i**

### CODE REVIEW

File: src/controllers/caseStudiesController.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { caseStudiesServices } = require('../services/caseStudiesServices');
```
Comment (rohini-senthil) [inline src/controllers/caseStudiesController.js:1]: Did you guys use camel case for file names during tech uni as well? (https://stackoverflow.com/questions/18927298/node-js-project-naming-conventions-for-files-folders)

File: src/services/caseStudiesServices.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { CaseStudies } = require('../models');
```
Comment (rohini-senthil) [inline src/services/caseStudiesServices.js:1]: Please include test cases in the PR

File: src/controllers/caseStudiesController.js (hunk 1 above)
Comment (rohini-senthil) [inline src/controllers/caseStudiesController.js:1]: Please add test cases for this controller as well

File: src/services/caseStudiesServices.js (hunk 2 above)
Comment (Cosmoboi-i) [reply src/services/caseStudiesServices.js:1]: I have included the tests. Will open a pull request again

**Pull request closed without merging**